airbyte-integrations/connectors/source-stripe/source_stripe/spec.yaml

```diff
@@ -0,0 +1,47 @@
+documentationUrl: https://docs.airbyte.io/integrations/sources/stripe
+connectionSpecification:
+  $schema: http://json-schema.org/draft-07/schema#
```

Comment:
feels a _little_ odd to be doing this in a yaml file, but I think it's still right?